### PR TITLE
Ci build updates

### PR DESCRIPTION
**Pull request by victorporof**
I've been running into the exact build problems described in https://github.com/bschwind/yoga-rs/issues/24 in my projects' CI. In the hopes of fixing things, I've updated the build dependencies to the latest version and added CI support to this repo as well.

If you merge this PR, you should turn on builds for this project in your Travis account and update the README badge.

### CODE REVIEW

**Comment by victorporof** [comment]
Can confirm this fixes build issues for me on linux CI.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  allow_failures:
+    - rust: nightly
+notifications:
+  email: false
+before_script:
+  - export PATH=$PATH:~/.cargo/bin
+script:
+  - cargo build --verbose
+  - cargo test --verbose
+after_success:
```
Comment (bschwind) [inline .travis.yml]: Could we remove this section? I'd prefer to just let crates.io handle the documentation (of which there is very little currently 😅 ). It also references rsx_parser currently and is passing a github token to a script I don't necessarily trust. I'd rather just avoid any issues here and let crates.io take care of the docs.

Comment (victorporof) [reply .travis.yml]: Sure thing. I was thinking you'd change these yourself, along with the readme badge, but I'll remove these.

Comment (bschwind) [reply .travis.yml]: Ah gotcha, yeah I'll change the badge and omit the `after_success` part of CI. Thanks!

**Comment by victorporof** [comment]
Done!

**Comment by bschwind** [comment]
LGTM!

**Pull request merged**